modules/efolder_vbms/lib/efolder_vbms.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'efolder_vbms/engine'
+
+module EfolderVBMS
+  def list_documents
+    if !Rails.env.development?
+      c = VBMS::Client.from_env_vars(env_name: Settings.vbms.env)
+      #ssn
+      req = VBMS::Requests::FindDocumentSeriesReference.new(ssn)
+      res = c.send_request(req)
+      #res = [[OpenStruct], [OpenStruct]]
+      dl = []
+      res.each do |d|
+          if d[0].doc_type == '184' or d[0].doc_type == '339'
+            dl << d[0]
+          end
+      end
+
+      dl = dl.sort_by{ |i| i.upload_date}
```

Comment:
Make sure you do a `.reverse` on these so they are sorted from most to least recent